AndroidAnnotations/androidannotations/src/main/java/org/androidannotations/handler/FragmentArgHandler.java

```diff
@@ -113,5 +113,12 @@ private void createBuilderInjectionMethod(Element element, EFragmentHolder holde
 		JVar arg = method.param(paramClass, fieldName);
 		method.body().invoke(builderArgsField, bundleHelper.getMethodNameToSave()).arg(argKeyStaticField).arg(arg);
 		method.body()._return(_this());
+
+		String docComment = processingEnv.getElementUtils().getDocComment(element);
+		if (docComment != null) {
+			method.javadoc().append(docComment.replaceAll("\r", "").trim());
```

Comment:
i assume you ask for the `.replaceAll("\r", "")`. I removed the `\n` because it breaks the output.

e.g.
```
/**
 * Hello
World
 *
 */
```
instead of
```
/**
 * Hello
 * World
 *
 */
```